src/main/webapp/discussion.js

```diff
@@ -22,13 +22,39 @@ const ATTR_ID = 'key-id';
 const ELEMENT_DISCUSSION = document.querySelector('#discussion-comments');
 const ELEMENT_POST_TEXTAREA = document.querySelector('#post-textarea');
 
+const TEMPLATE_COMMENT = document.querySelector('#comment-template');
+
+const SLOT_HEADER = 'header';
+const SLOT_CONTENT = 'content';
+const SLOT_REPLIES = 'replies';
+
+const SELECTOR_SHOW_REPLY = '#show-reply';
+const SELECTOR_REPLY_FORM = '#reply-form';
+const SELECTOR_CANCEL_REPLY = '#cancel-reply';
+const SELECTOR_POST_REPLY = '#post-reply';
+const SELECTOR_REPLY_TEXTAREA = '#reply-textarea';
+
 /**
  * Loads the lecture disucssion.
  */
 async function intializeDiscussion() {
   await loadDiscussion();
 }
 
+/**
+ * Posts a new comment using the main post textarea.
+ */
+async function postNewComment() {
+  postAndReload(ELEMENT_POST_TEXTAREA);
+}
+
+/**
+ * Posts a reply to a comment.
+ */
+async function postReply(textarea, parentId) {
```

Comment:
seems textarea should be textArea

Reply:
I think that would be confusing, as the element tag is always written as `textarea` : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

Reply:
I think i'll change it to inputField, as it doesn't need to be a textarea

Reply:
Done